### DISCUSSION TITLE
Does anyone have a typst slide custom format that they can share?

### DISCUSSION THREAD

**Discussion by mpf** [quarto-dev/quarto-cli#8641] [Q&A]
### Description

I discovered these typst [custom formats](https://quarto.org/docs/output-formats/typst-custom.html), which are great. Does anyone have one for slides that they might share?  Thanks!

**Comment by mcanouil** [comment]
You should look online for Typst templates and try to adapt as a Quarto extension format following the few examples you linked.

If someone make something, he/she will unlikely share to your thread.
Instead:
- https://quarto.org/docs/extensions/
- https://github.com/mcanouil/awesome-quarto?tab=readme-ov-file#extensions (https://m.canouil.dev/quarto-extensions/)
